app/models/spree/sale_image/configuration/active_storage.rb

```diff
@@ -12,9 +12,9 @@ module ActiveStorage
 
           def self.styles
             @styles ||= {
-              mini: '100x100>',
-              small: '240x240>',
-              medium: '600x600'
+              mini: '240v60>',
```

Comment:
Let's use the image scale in factors of 1x, 2x, and 3x (optional) and min height is 150px
- 600 x 150 (1x)
- 1200 x 300 (2x)
- ...